tests/resources/Group23-VIC-Machine-Service-Util.robot

```diff
@@ -14,21 +14,31 @@
 
 *** Settings ***
 Documentation    This resource contains keywords which are helpful for using curl to test the vic-machine API.
-
+Library  Process
 
 *** Variables ***
-${HTTP_PORT}     1337
-${HTTPS_PORT}    31337
-
 ${RC}            The return code of the last curl invocation
 ${OUTPUT}        The output of the last curl invocation
 ${STATUS}        The HTTP status of the last curl invocation
 
 
 *** Keywords ***
 Start VIC Machine Server
-    Start Process    ./bin/vic-machine-server --port ${HTTP_PORT} --scheme http    shell=True    cwd=/go/src/github.com/vmware/vic
-
+    ${http_port}=    Get HTTP Port
+    Set Suite Variable    ${HTTP_PORT}    ${http_port}
+    ${handle}=    Start Process    ./bin/vic-machine-server --port ${HTTP_PORT} --scheme http    shell=True    cwd=/go/src/github.com/vmware/vic
+    Process Should Be Running    ${handle}
+    [Return]    ${handle}
```

Comment:
Can we store the handle in a suite variable here to avoid needing to do that in each suite?

Reply:
@zjs I remember there being a reason I did it this way, but I'll change it once the build passes for this first PR run.